tests/fixture/tmpnet/network.go

```diff
@@ -888,6 +864,129 @@ func (n *Network) GetPluginDir() (string, error) {
 	return n.DefaultFlags.GetStringVal(config.PluginDirKey)
 }
 
+// GetGenesisFileContent returns the base64-encoded JSON-marshaled
+// network genesis.
+func (n *Network) GetGenesisFileContent() (string, error) {
+	bytes, err := json.Marshal(n.Genesis)
+	if err != nil {
+		return "", fmt.Errorf("failed to marshal genesis: %w", err)
+	}
+	return base64.StdEncoding.EncodeToString(bytes), nil
+}
+
+// GetSubnetConfigContent returns the base64-encoded and
+// JSON-marshaled map of subnetID to subnet configuration.
+func (n *Network) GetSubnetConfigContent() (string, error) {
+	subnetConfigs := maps.Clone(n.PrimarySubnetConfigs)
```

Comment:
`n.PrimarySubnetConfigs` seems to never be set, I suppose this is coming in an upcoming PR? Might be worth moving that field to the relevant PR where it's starting to be used? 🤔